src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = [...document.querySelectorAll('.population')].map((el) => {
+  return +el.innerText.replace(/,/g, '');
+});
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+const totalPopulation = populations.reduce((a, b) => a + b);
+const averagePopulation = totalPopulation / 10;
```

Comment:
The divisor '10' seems to be a hard-coded value. It would be better to use 'populations.length' to ensure the calculation is accurate regardless of the array's length.